src/operator/windowTime.ts

```diff
@@ -147,52 +163,35 @@ class WindowTimeSubscriber<T> extends Subscriber<T> {
     return window;
   }
 
-  closeWindow(window: Subject<T>) {
+  closeWindow(window: Subject<T>): void {
     window.complete();
     const windows = this.windows;
     windows.splice(windows.indexOf(window), 1);
   }
 }
 
-interface TimeSpanOnlyState<T> {
-  window: Subject<any>;
-  windowTimeSpan: number;
-  subscriber: WindowTimeSubscriber<T>;
-}
-
-function dispatchWindowTimeSpanOnly<T>(this: Action<TimeSpanOnlyState<T>>, state: TimeSpanOnlyState<T>) {
+function dispatchWindowTimeSpanOnly<T>(this: Action<TimeSpanOnlyState<T>>, state: TimeSpanOnlyState<T>): void {
   const { subscriber, windowTimeSpan, window } = state;
   if (window) {
-    window.complete();
+    subscriber.closeWindow(window);
```

Comment:
Yeah, we might want to. This is a commit to merge in a fix to production code... so we might want to make sure the changes are isolated this time.